config/opensearch_dashboards.yml

```diff
@@ -379,3 +379,7 @@
 
 # Set the value to true to enable the new UI for savedQueries in Discover
 # data.savedQueriesNewUI.enabled: true
+workspace.enabled: true
+opensearch.ignoreVersionMismatch: true
+data_source.enabled: true
+data.savedQueriesNewUI.enabled: true
```

Comment:
can we remove these? previously these additions were causing all of our tests to fail